funnel/views/schedule.py

```diff
@@ -109,7 +109,7 @@ def schedule_data(
         else:
             data[day] = {}
     schedule = []
-    for day in sorted(data):
+    for day in sorted(data, reverse=True):
```

Comment:
This will show future dates before present day. What we need is to hide past dates until the user asks to see the past.